BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("//bazel:copts.bzl", "entt_copts")
+
+cc_library(
+    name = "entt",
```

Comment:
If someone where to use `EnTT` with bazel it would look something like this:

```python
cc_binary(
    name = "example",
    srcs = ["example.cc"],
    deps = ["@com_github_skypjack_entt//:entt"],
)
```